common/menu.c

```diff
@@ -267,10 +290,24 @@ void menuStartup() {
 
     folder_icon_small = downscaleImg(folder_icon_bin, 256, 256, 140, 140, IMAGE_MODE_RGB24);
     invalid_icon_small = downscaleImg(invalid_icon_bin, 256, 256, 140, 140, IMAGE_MODE_RGB24);
+    if(themeGlobalPreset == THEME_PRESET_DARK)
+        theme_icon_small = downscaleImg(theme_icon_dark_bin, 256, 256, 140, 140, IMAGE_MODE_RGB24);
+    else
+        theme_icon_small = downscaleImg(theme_icon_light_bin, 256, 256, 140, 140, IMAGE_MODE_RGB24);
     computeFrontGradient(themeCurrent.frontWaveColor, 280);
     //menuCreateMsgBox(780, 300, "This is a test");
 }
 
+void themeMenuStartup() {
+    if(hbmenu_state != HBMENU_DEFAULT) return;
+    hbmenu_state = HBMENU_THEME_MENU;
+    char tmp_path[PATH_MAX];
+
+    snprintf(tmp_path, sizeof(tmp_path)-1, "%s%s%s%s%s%s",DIRECTORY_SEPARATOR, "config", DIRECTORY_SEPARATOR, "nx-hbmenu" , DIRECTORY_SEPARATOR, "themes");
```

Comment:
It doesn't look like this would work on hardware...

Reply:
pc-build that is.

Reply:
lol, i uses the directory_seperators so that it would though